frontend/src/views/WheresMyGeneV2/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -218,7 +218,11 @@ function getNewSelectedFilters({
     Object.entries(tissues ?? {}).map(([id, tissue]) => [tissue.name, id])
   );
 
-  const allTissueNames = Object.keys(tissueIdsByName);
```

Comment:
`allTissueNames = Object.keys(tissueIdsByName);` was returning an empty array, since `Object.keys()` doesn't work with Map! TIL 💡 

This fixes the bug where share link tissue with tissue name won't get populated in the app state